floor/floor/processor/simple_step.py

```diff
@@ -26,20 +24,13 @@ class SimpleStep(Base):
 
     def __init__(self, **kwargs):
         super(SimpleStep, self).__init__(**kwargs)
-        self.hue = 1.0
-        self.saturation = 1.0
-        self.value = 1.0
-        self.red = COLOR_MAXIMUM
-        self.green = 0
-        self.blue = 0
 
-    @clocked(frames_per_second=24)
     def get_next_frame(self, context):
         weights = context.weights
         pixels = [(0, 0, 0)] * 64
 
         for idx in range(64):
             if weights[idx] > 0:
-                pixels[idx] = self.hsv_to_rgb([self.hue, self.saturation, self.value])
+                pixels[idx] = self.hsv_to_rgb([self.HUE, self.SAT, self.VAL])
```

Comment:
This is now routed correctly and can be controlled via midi.  This is probably a good overlay for letting people see steps easily.